sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensor.java

```diff
@@ -72,40 +68,55 @@ public CxxCoverageSensor(Settings settings, FileSystem fs, ProjectReactor reacto
     parsers.add(new VisualStudioParser());
   }
 
+  @Override
+  public boolean shouldExecuteOnProject(Project project) {
+    return fs.hasFiles(fs.predicates().hasLanguage(CxxLanguage.KEY));
```

Comment:
This is a copy of the already existing CxxReportSensor.java / shouldExecuteOnProject: means no change to old version.